productService/serverless.yml

```diff
@@ -0,0 +1,28 @@
+service: my-store-app-backend
+
+frameworkVersion: "3"
+
+plugins:
+  - serverless-webpack
+
+provider:
+  name: aws
+  runtime: nodejs14.x
```

Comment:
NIT: try to use as latest as possible versions to be able to use new features and fixes